src/eval-frame/components/reps/tiny-reps.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import styled from "react-emotion";
+
+const numberColor = "rgb(28, 0, 207)";
+const boolColor = numberColor;
+const nullUndefinedColor = "#808080";
+const classInfoColor = "#808080";
+const stringColor = "rgb(196, 26, 22)";
+const separatorColor = "black";
+const dateColor = stringColor;
+const symbolColor = stringColor;
+const errorColor = "";
+const functionColor = "rgb(28, 30, 207)";
+const ellipsisColor = "#111 ";
+
+const RepBaseStyledSpan = color => styled("span")`
+  color: ${color};
+  font-family: Menlo, monospace;
+  font-size: 12px;
+  line-height: 14px;
+`;
+
+const tinyStringValueWithColor = color => {
+  const ValueSpan = RepBaseStyledSpan(color);
+  return x => <ValueSpan>{x.stringValue}</ValueSpan>;
+};
+
+const ClassInfoSpan = RepBaseStyledSpan(classInfoColor);
+
+const objectRep = obj => (
+  <ClassInfoSpan>
+    {obj.class}({obj.size})
+  </ClassInfoSpan>
+);
+
+const Sep = RepBaseStyledSpan(separatorColor);
+const Ell = RepBaseStyledSpan(ellipsisColor);
+const StringSpan = RepBaseStyledSpan(stringColor);
+const quotedStringRep = (lQuote, rQuote) => obj => (
+  <span>
+    <Sep>{lQuote}</Sep>
+    <StringSpan>{obj.stringValue}</StringSpan>
+    {obj.isTruncated ? <Ell>⋯</Ell> : ""}
+    <Sep>{rQuote}</Sep>
+    {obj.isTruncated ? <ClassInfoSpan>({obj.size})</ClassInfoSpan> : ""}
+  </span>
+);
+
+const stringRep = quotedStringRep("“", "”");
```

Comment:
I'm not a fan of using "typographic" quotes in this context, because it means you can't copy and paste the quotes into code.

Reply:
sure, that makes sense